dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/resource/set/DonorSetResource.java

```diff
@@ -98,6 +98,7 @@ public UploadedDonorSet processDonorSet(
       @ApiParam(value = "Validation") @QueryParam("validationOnly") @DefaultValue("false") boolean validationOnly,
       @ApiParam(value = "External Repository") @QueryParam("externalRepo") @DefaultValue("false") boolean externalRepo) {
 
+    checkRequest(donorIds == null, "DonorIds cannot be null.");
```

Comment:
Can this ever be null? In my experience it would be always empty. However, this wasn't for `@FormParam`, only `@QueryParam`